lib/generator/repository.rb

```diff
@@ -30,6 +30,11 @@ def create_tests_file
         template: tests_template.to_s,
         values: template_values
       )
+      display_result
+    end
+
+    def display_result
+      $stdout.puts "Generated #{exercise_name} tests version #{version}"
```

Comment:
It will work without it, but I consider using a global to be a smell and like to make it explicit that's what's happening.

This has inspired me to refactor to eliminate it.